app/assets/javascripts/initializers/initializeCommentsPage.js.erb

```diff
@@ -198,13 +198,6 @@ function handleCommentSubmit(event) {
       if (response.status === 200) {
         response.json().then(function (newComment) {
           var newNode = document.createElement('div');
-          if (newComment.status == "comment already exists") {
```

Comment:
these just go in the else now. See line 198 above and 

https://github.com/thepracticaldev/dev.to/blob/d4c1b44a21eb4e767eb012e075edc5e72a8ca1c6/app/assets/javascripts/initializers/initializeCommentsPage.js.erb#L249-L255